frontend/src/components/organisms/MatchesList/MatchStatus.tsx

```diff
@@ -0,0 +1,6 @@
+export enum MatchStatus {
+    FINISHED= "Zakończony",
+    CALLED= "Podczas Gry",
```

Comment:
To raczej nazywamy "Wywołany". Przynajmniej takie slownictwo funkcjonuje wsrod "pilkarzykowcow" :)

Reply:
Tutaj tez jakby formatowania nie bylo.

Reply:
Ok, poprawię!